src/networks/abc/chainparamsconstants.cpp

```diff
@@ -5,8 +5,8 @@
 #include <chainparamsconstants.h>
 
 namespace ChainParamsConstants {
-    const BlockHash MAINNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("0000000000224e1e192ef31b0da7a5697b8f272b9b9fd66695f61add3b2d3216");
-    const uint256 MAINNET_MINIMUM_CHAIN_WORK = uint256S("000000000000000000000000000000000000000000000000046b12db9e6b24ef");
+    const BlockHash MAINNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("000000000006a214caf1b62fdc5fa9746aa90d28878868561dcc1741da19eade");
+    const uint256 MAINNET_MINIMUM_CHAIN_WORK = uint256S("000000000000000000000000000000000000000000000000243805a5b0aed85e");
```

Comment:
eCash doesn't have the same minimum chain work associated with the assume valid block hash. Need to figure out what's going on.